components/editor/modules/dynamiccomponent/Placeholder.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { css } from 'glamor'
+import { Interaction, colors } from '@project-r/styleguide'
+
+const styles = {
+  box: css({
+    background: colors.primaryBg,
+    pointerEvents: 'none',
+    padding: '5px 7px'
+  })
+}
+
+const Placeholder = identifier => () => {
```

Comment:
Let's make this a regular React component:
```suggestion
const Placeholder = ({ identifier }) => {
```
The factory creates unnecessary re-renders because React can't recognise that it's the same component. Btw. if we were to do a factory we should call it something like `createPlaceholder` or even `createPlaceholderComponent` to make it easier to read and understand.

Reply:
I suggest adding `identifier` as default prop in the styleguide.